src/FormCollection/README.md

```diff
@@ -0,0 +1,153 @@
+# UX Form Collection
+
+Symfony UX Form collection is a Symfony bundle providing light UX for collection
+in Symfony Forms.
+
+## Installation
+
+UX Form Collection requires PHP 7.2+ and Symfony 4.4+.
+
+Install this bundle using Composer and Symfony Flex:
+
+```sh
+composer require symfony/ux-form-collection
+
+# Don't forget to install the JavaScript dependencies as well and compile
+yarn install --force
+yarn encore dev
+```
+
+Also make sure you have at least version 2.0 of [@symfony/stimulus-bridge](https://github.com/symfony/stimulus-bridge)
+in your `package.json` file.
+
+You need to select the right theme from the one you are using :
+```yaml
+# config/packages/twig.yaml
+twig:
+  # For bootstrap for example
+  form_themes: ['@FormCollection/form_theme_div.html.twig']
+```
+You have 2 different themes :
+- `@FormCollection/form_theme_div.html.twig`
+- `@FormCollection/form_theme_table.html.twig`
+
+[Check the Symfony doc](https://symfony.com/doc/4.4/form/form_themes.html) for the different ways to set themes in Symfony.
+
+## Usage
+
+The most common usage of Form Collection is to use it as a replacement of
+the native CollectionType class:
+
+```php
+// ...
+use Symfony\UX\FormCollection\Form\UXCollectionType;
+
+class BlogFormType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            // ...
+            ->add('comments', UXCollectionType::class, [
+                // ...
+                'button_add' => [
+                     // Default text for the add button
+                    'text' => 'Add',    
+                    // Add HTML classes to the add button
+                    'class' => 'btn btn-outline-primary'
+                ],
+                'button_delete' => [
+                    // Default text for the delete button
+                    'text' => 'Remove',    
+                    // Add HTML classes to the add button
+                    'class' => 'btn btn-outline-secondary'
+                ],
```

Comment:
I'm not sure about me but have you tried with 2 ButtonType in the Form when the options "allow_add" & "allow_delete" are true?
This would allow to modify their appearance via the twig blocks

Reply:
@maxhelias allow_add is a boolean type from the original [FormType](https://symfony.com/doc/current/reference/forms/types/collection.html#allow-add).